Mobile/starter_project_dartsmiths/.gitignore

```diff
@@ -42,3 +42,6 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
+
+.fake
+.dart_tool/package_config.json
```

Comment:
and since you have already commited changes in that folder (and .vscode folder too, and android folder too), you will have to add another commit to revese those changes before adding them to git ignore

Reply:
I don't think the android folder should be ignored. Later on there will be build changes and config changes to those files, I agree on ignoring the .dart_tools and .vs_code folders though.